fbw-common/src/systems/datalink/atc/src/DigitalOutputs.ts

```diff
@@ -119,6 +121,21 @@ export class DigitalOutputs {
     }
   }
 
+  public sendRmpFrequency(frequency: number): void {
+    const frequencyWord = Arinc429Register.empty();
```

Comment:
It would be better to keep this around as a static member rather than needing to allocate a new one each time. Then publish to LVar instead of event bus.